### ISSUE TITLE
Get WebRTC error 1006 after updating to bbb 2.3-alpha-5 

### ISSUE THREAD

**Issue by jdepi** [bigbluebutton/bigbluebutton#11278]
**Describe the bug**
After updating to BigBlueButton 2.3-alpha-5 I get this webrtc error 1006 in Firefox 85.0 64 Bit and Chromium Version 84.0.4147.135 (Developer Build) built on Ubuntu focal/20.04, running on Linuxmint ulyssa/20.1 (64-bit)

**To Reproduce**
1. Via greenlight open a room
2. click on mic button
3. voila - there it appears after approx. 20 seconds
4. (turn log shows no connection)

**Expected behavior**
No error in html5 frontend

**Actual behavior**
html5 client displays a webrtc error number 1006

**Screenshots**

**Desktop (please complete the following information):**
 - LinuxMint 20.1 based on Ubuntu 20.04
 - Firefox 85 and chromium 84

**Comment by Beiri22** [comment]
Same thing to me:

1.) When I enter with mic: The connection to echo test times out: Error 1006 ([21:22:29:0910] ERROR: clientLogger: Audio error - errorCode=1006, cause=Call timed out on start after 20s https://bbbb.beiri22.de/html5client/1/3db6e0bbbc534f411771528262979c2d67eb22de.js?meteor_js_resource=true:353:1107414
)
2.) When trying to enter "only to listen": nothing happens when clicking on that button. (nothing in console log, too)

Camera seems to work.

**Comment by ffdixon** [comment]
Can you confirm that you upgraded by re-running the `bbb-install.sh` script (this is how to upgrade to the latest build of 2.3-dev)?  

Also, can you confirm you are running 2.3-alpha-5 on Ubuntu 18.04.

**Comment by Beiri22** [comment]
I had installed 2.3a4 by the Install Script,so yes or IS running on 18.04. ... But did the update by using apt update and upgrade. I will check what happens...

**Comment by jdepi** [comment]
I used 'apt upgrade' on 18.04 and 2.3-alpha-4. Are there any params required for the update with bbb-install.sh?

**Comment by Beiri22** [comment]
I confirm, using bbb-install the error does not occur.

As the release information specified: "You may need to update the url manually if you have a pre-existing 2.3 installation." -> I thought that would indicate a regular apt update + some manual work might be ok. Does bbb-install.sh leave all settings of a previous installation untouched?

**Comment by basisbit** [comment]
You both apparently never have upgraded a production-server of BBB before - how come you started using an unstable alpha version as your first BigBlueButton server version? Do we miss some warning labels somewhere?

**Comment by jdepi** [comment]
Hi basisbit, setting up an unstable alpha server was a conscious decision. Yes, I'm missing a warning level if apt upgrade should not be used for upgrading bbb.

**Comment by ffdixon** [comment]
Hi @jdepi, the upgrade via `apt-get update` is not sufficient to upgrade to the latest version of 2.3-dev.  Instead, we recommend re-running the initial `bbb-install.sh` (with the same parameters) and it will pull down the latest version of 2.3-dev and re-apply the needed configuration changes to make sure it works (and if doesn't, let us know).

**Comment by Beiri22** [comment]
> You both apparently never have upgraded a production-server of BBB before - how come you started using an unstable alpha version as your first BigBlueButton server version? Do we miss some warning labels somewhere?

No need to get rude. I am user of a 2.2-instance (university) and am interested in changes and development - so I set up a vm with the 2.3-a just to have a look.

**Comment by basisbit** [comment]
@Beiri22 sorry, I did not intend this to be rude. I now edited my previous message to reflect that I was just curious for why so many people in the forums and here on GitHub lately use bbb2.3-alpha as their first BBB server version.

**Comment by Beiri22** [comment]
Thanks for clarifying. I also edited my post accordingly. I am planning a personal instance of bbb in the future, which might be of the new version. So from time to time I'm doing some testing. I also have a someway-hacked bbb-2.2-on-demand that is hosted on hetzner and can be removed / rebuild on base of an image with a simple shellscript, so that I only pay the time I use it.

**Comment by qdzmc** [comment]
bbb-conf --setip www.xxx.xxx

**Comment by michi-80337** [comment]
I ran into the same problem by accident with BigBlueButton Server 2.3.13 (2444) when I did an `apt-get update; apt-get upgrade` to install security updates for the Ubuntu distribution. As there were some days between the update and the detection of the problem, I can't remember if there was a warning about conflicting config file modifications or not.

Calling `bbb-conf --check` finally hinted me to `/etc/bigbluebutton/nginx/sip.nginx`. All I had to do, was to repair the proxy_pass statement in the file, and to restart the nginx service. Hence its always a good idea to do a bbb-conf --check after a software update. A `bbb-conf --set-ip` might have worked, too.

It might be an idea to add a hint into the [troubleshooting documentation](https://docs.bigbluebutton.org/support/troubleshooting.html#webrtc-errors-1001-1002) at the bullet point "1006: Call timed out".
